src/filesystem.cpp

```diff
@@ -510,14 +509,14 @@ std::string get_next_filename(const std::string& name, const std::string& extens
 
 static bfs::path user_data_dir, user_config_dir, cache_dir;
 
-static const std::string get_version_path_suffix(const version_info& version)
+std::string get_version_path_suffix(const version_info& version)
 {
 	std::ostringstream s;
 	s << version.major_version() << '.' << version.minor_version();
 	return s.str();
 }
 
-static const std::string& get_version_path_suffix()
+std::string& get_version_path_suffix()
```

Comment:
Either this needs to be const, or it needs to lose the behavior of caching a static string.

Reply:
Updated.